packages/engine/src/proto.rs

```diff
@@ -272,9 +272,9 @@ impl<'de> Deserialize<'de> for MetricObjective {
 pub struct SimpleExperimentConfig {
     /// The experiment name
     pub experiment_name: ExperimentName,
-    /// The properties changed for each simulation run
+    /// The global properties changed for each simulation run
     #[serde(rename = "changedProperties")]
```

Comment:
Does this also have to be renamed?
```suggestion
    #[serde(rename = "changedGlobals")]
```

Reply:
At the moment we've decided it's safer to leave it as properties as that's what's expected by the API when we replace Cloud.

Reply:
We can't rename this now, because it's based on what the HASH API expects from cloud.